README.md

```diff
@@ -29,38 +30,42 @@
 
 ## Data representation
 
-    BSON                Elixir
-    ----------          ------
-    double              0.0
-    string              "Elixir"
-    document            [{"key", "value"}] | %{"key" => "value"} (1)
-    binary              %BSON.Binary{binary: <<42, 43>>, subtype: :generic}
-    object id           %BSON.ObjectId{value: <<...>>}
-    boolean             true | false
-    UTC datetime        %DateTime{}
-    null                nil
-    regex               %BSON.Regex{pattern: "..."}
-    JavaScript          %BSON.JavaScript{code: "..."}
-    integer             42
-    symbol              "foo" (2)
-    min key             :BSON_min
-    max key             :BSON_max
-
-1) Since BSON documents are ordered Elixir maps cannot be used to fully represent them. This driver chose to accept both maps and lists of key-value pairs when encoding but will only decode documents to maps. This has the side-effect that the information about order of keys in a BSON document is lost when it's decoded. Additionally the driver will accept both atoms and strings for document keys but will only decode to strings.
-
-2) BSON symbols can only be decoded.
+| BSON             | Elixir
++------------------+-------
+| double           | `0.0`
+| string           | `"Elixir"`
+| document         | `[{"key", "value"}]` | `%{"key" => "value"}` ¹
+| binary           | `%BSON.Binary{binary: <<42, 43>>, subtype: :generic}`
+| object id        | `%BSON.ObjectId{value: <<...>>}`
+| boolean          | `true` | `false`
+| UTC datetime     | `%DateTime{}`
+| null             | `nil`
+| regex            | `%BSON.Regex{pattern: "..."}`
+| JavaScript       | `%BSON.JavaScript{code: "..."}`
+| integer          | `42`
+| symbol           | `"foo"` ²
+| min key          | `:BSON_min`
+| max key          | `:BSON_max`
+
+¹ Since BSON documents are ordered Elixir maps cannot be used to fully
+represent them. This driver chose to accept both maps and lists of key-value
+pairs when encoding but will only decode documents to maps. This has the
+side-effect that the information about order of keys in a BSON document is lost
+when it's decoded. Additionally the driver will accept both atoms and strings
+for document keys but will only decode to strings.
+
+² BSON symbols can only be decoded.
 
 ## Usage
 
 ### Installation:
 
-Add mongodb to your mix.exs `deps` . Mongodb supports the same pooling libraries db_connection does (currently: no pooling, poolboy, and sbroker). If you want to use poolboy as pooling library you should set up your project like this:
```

Comment:
This is removed as it is no longer true as we currently use `db_connection` 2.x which has custom pool implementation and do not support neither Poolboy nor Sbroker.